starter-project-web/store/lydia.js

```diff
@@ -0,0 +1,44 @@
+export const state = () => ({
+  posts: [
+  ]
+})
+
+export const mutations = {
+  setPosts (state, posts) {
+    state.posts = posts
+  },
+  addPost (state, post) {
+    state.posts.push(post)
+  },
+  editPost (state, post) {
+    state.posts = state.posts.filter(item => item.id !== post.id)
+    state.posts.push(post)
+  },
+  deletePost (state, id) {
+    state.posts = state.posts.filter(post => post.id !== id)
+  }
+}
+
+export const actions = {
+  async fetchPosts ({ commit }) {
+    const response = await this.$axios.get('http://jsonplaceholder.typicode.com/posts?userId=1')
```

Comment:
change http to https. the fetched posts are not visible on the deployed page with http